pyndl/io.py

```diff
@@ -153,3 +154,39 @@ def events_from_list(lst):
         if isinstance(outcomes, str):
             outcomes = outcomes.split('_')
         yield (cues, outcomes)
+
+
+def safe_write_path(path, template='{path.stem}-{counter}{path.suffix}'):
+    """
+    Create a file path to avoid overwriting existing files.
+    Returns the original path if it does not exist or
+    an incremented version according to the template.
+    
+    The default template creates filenames like
```

Comment:
I think this should be called rather "this function creates filenames like" as the template itself does not. Probably generally worth mentioning here that the returned path is either the path or following the template?

Reply:
Thanks, it's now more explicit.